Doc/library/profile.rst

```diff
@@ -577,23 +584,20 @@ implementations.
 Limitations
 ===========
 
-One limitation has to do with accuracy of timing information. There is a
-fundamental problem with deterministic profilers involving accuracy.  The most
-obvious restriction is that the underlying "clock" is only ticking at a rate
-(typically) of about .001 seconds.  Hence no measurements will be more accurate
-than the underlying clock.  If enough measurements are taken, then the "error"
-will tend to average out. Unfortunately, removing this first error induces a
-second source of error.
+To avoid infinite recursion, :mod:`profile` and :mod:`cProfile` do not work
+inside the trace function set by :func:`sys.settrace` or :func:`sys.setprofile`.
 
-The second problem is that it "takes a while" from when an event is dispatched
+It "takes a while" from when an event is dispatched
 until the profiler's call to get the time actually *gets* the state of the
 clock.  Similarly, there is a certain lag when exiting the profiler event
 handler from the time that the clock's value was obtained (and then squirreled
 away), until the user's code is once again executing.  As a result, functions
 that are called many times, or call many functions, will typically accumulate
-this error. The error that accumulates in this fashion is typically less than
-the accuracy of the clock (less than one clock tick), but it *can* accumulate
-and become very significant.
```

Comment:
I'd leave this?

Reply:
The sentence `the error that accumulates in this fashion is typically less than
the accuracy of the clock (less than one clock tick)` is incorrect now, as the accuracy of the clock has improved a lot for the last .. I don't know how many years. This is definitely more significant than the clock accuracy issue. Do you want me to keep the "become very significant"?